lib/db/client-applications-store.js

```diff
@@ -27,6 +29,7 @@ const mapRow = row => ({
 const remapRow = (input, old = {}) => ({
     app_name: input.appName,
     updated_at: input.updatedAt,
+    created_at: input.createdAt,
```

Comment:
modifying the schema might, more easily lend itself to being a step after separation